04/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <meta name="description" content="Ejecicios de listas de el curso de HTML con recetas de comidas mexicanas"
+
+    <meta name="keywords" content="Ejecicios, listas, recetas, html, huevos, jamón"
+
+    <meta name="author" content="Jesús David Ceballos Parra"
```

Comment:
Are these all the meta tags we have seen? / ¿Son todas las meta que hemos visto?

Reply:
según yo si!